domains/alden.json

```diff
@@ -0,0 +1,14 @@
+{
+    "description": "Websitenya alden si dawg yang sangaddd keren",
+    "repo": "https://github.com/Schmeichel-web/aldenpbm.github.io",
+    "owner": {
+        "username":"Schmeichel-web",
+        "manager": "cupglassdev",
+        "email": "",
+        "discord": "footballfan0891",
+        "note":"this is my friend's website, because he dosent really understands about these thing, im managing it"
+    },
+    "record": {
+        "CNAME": "aldenpbm.github.io"
```

Comment:
I'm getting a 404 page, is there any content there?

Reply:
Yes, there is a content in there

Reply:
https://github.com/Schmeichel-web/aldenpbm.github.io